src/resolvers/account/index.ts

```diff
@@ -34,4 +35,18 @@ export const AccountResolver = {
       return { token };
     },
   },
+  Query: {
+    me: (_: never, {}, ctx: Context): Account => {
+      const data: Account = ctx.user.data;
+      const user = {
+        id: Buffer.from(data.id),
+        firstName: data.firstName,
+        lastName: data.lastName,
+        emailAddress: data.emailAddress,
+        createdAt: data.createdAt,
+        updatedAt: data.updatedAt,
+      };
```

Comment:
Put the `Buffer.from(data.id)` conversion in your private directive.
Then in here, you can just return the account.
```typescript
me: (_: unknown, _args: unknown, ctx: Context): Account => {
  return ctx.user.data;
}
```